test/common.js

```diff
@@ -64,18 +66,23 @@ function rmdirSync(p, originalEr) {
 }
 
 exports.refreshTmpDir = function() {
+  try {
+    fs.accessSync(testRoot, fs.W_OK);
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      fs.mkdirSync(testRoot);
```

Comment:
I'm not convinced about this, it really needs a mkdirp because there's zero guarantee that the parent will exist, perhaps the user should get the error if they have tried to supply a non-existent directory?